### PR TITLE
Fixing sandbox unbound variable error

### PR DESCRIPTION
**Pull request by radoslawc**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>

kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:
When using set -o nounset in Bash scripts variables have to have value, here the default will be all to execute all tags if not overriden 
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: e2e/provision/install_sandbox.sh [hunk 1]
```diff
@@ -75,7 +75,7 @@ fact_caching_connection = /tmp
 EOT
 
 # Management cluster creation
-ansible_cmd="$(command -v ansible-playbook) -i 127.0.0.1, playbooks/cluster.yml --tags ${ANSIBLE_TAG} "
+ansible_cmd="$(command -v ansible-playbook) -i 127.0.0.1, playbooks/cluster.yml --tags ${ANSIBLE_TAG:-all} "
```
Comment (vjayaramrh) [inline e2e/provision/install_sandbox.sh:78]: @radoslawc FYI I believe that it is taken care at https://github.com/nephio-project/test-infra/blob/main/e2e/provision/init.sh#L84-L88 .. is that insufficient?

Comment (radoslawc) [reply e2e/provision/install_sandbox.sh:78]: @vjayaramrh yes, that's correct, but it has no way of knowing that it's being sourced by init.sh where this variable gets populated and when the script has set -o nounset it fails on unbound variable which we see in logs. It exits with status 1 failing whole job.

**Comment by radoslawc** [comment]
e2e tests are failing due to ongoing R3 effort, change has no impact on them
/override e2e-free5gc-ubuntu-focal
/override e2e-oai-ubuntu-jammy
/override  e2e-oai-fedora-34
/override e2e-free5gc-fedora-34

**Review by vjayaramrh — APPROVED** [review]
/lgtm

**Pull request merged**